docs/fsharp/using-fsharp-on-azure/package-management.md

```diff
@@ -11,99 +11,85 @@ Obtaining packages for Azure development is easy when you use a package manager.
 
 ## Using Paket
```

Comment:
Given the new focus around scripting,  I agree that we should kill this page.

Reply:
Okay thank you.